sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/MigrationGuide.md

```diff
@@ -91,3 +100,162 @@ AnswersOptions options = new AnswersOptions
 
 Response<AnswersResult> responseFollowUp = await client.GetAnswersAsync("{question}", project, options);
 ```
+
+### Authoring Client
+
+#### Authenticating authoring client
+
+Previously in `Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker`, you could create a `QnAMakerClient` along with `EndpointKeyServiceClientCredentials`:
+
+```C# Snippet:CognitiveServices_QnA_Maker_Snippets_MigrationGuide_CreateClient
+QnAMakerClient client = new QnAMakerClient(new ApiKeyServiceClientCredentials("{QnAMakerSubscriptionKey}"), new HttpClient(), false)
+{
+    Endpoint = "https://westus.api.cognitive.microsoft.com"
+};
+```
+
+Now in `Azure.AI.Language.QuestionAnswering.Projects`, you create a `QuestionAnsweringProjectsClient` along with `AzureKeyCredential` from the package `Azure.Core`:
+
+```C# Snippet:Language_QnA_Maker_Snippets_MigrationGuide_CreateClient
+Uri endpoint = new Uri("https://myaccount.api.cognitive.microsoft.com");
```

Comment:
We should use obvious placeholders like we do in existing samples e.g., `{questionanswering_uri}`.

Reply:
Done